packages/spaces/commands/vpn/update.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
```

Comment:
can you remove `co` here? I know it's used in other commands in this package but we'd like to get rid of it long term as node now supports `async/await` natively. I'll leave a comment below for the `run` function to show you what new stuff in v5 plugins should look like (until this can be migrated to a v7 oclif plugin)
```suggestion
```